app/models/ability.rb

```diff
@@ -11,6 +11,21 @@ def initialize(current_user, current_group=nil)
         person.groups.include?(current_group)
       end
 
+      can :manage, Person do |person|
```

Comment:
later, if we were worried about performance (which i'm not ATM), i'd wonder whether it was worth trying to optimize these sql queries a bit (since we hit them every time we try to perform this auth check, and in the case where a group is affiliated with many other groups, we might rack up a few). but we're not worried about that. so just flagging it for later so we think about it now and can remember if we need to. (also maybe worth looking at whether permission-checking is expensive on pages where we *know* we're currently having perf issues?)